### PR TITLE
Pass through all headers to client

### PR DESCRIPTION
**Pull request by brettshollenberger**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -130,7 +130,12 @@ http.createServer(function(req, res) {
 	});
 	req.on('end', function () {
 		request(options, function(err, response, body) {
-			res.setHeader('content-type', response.headers['content-type']);
+      var headerName;
```
Comment (mattpardee) [inline index.js:133]: Don't need this var declaration since you're defining it in the for loop below

Comment (brettshollenberger) [inline index.js:133]: Apologies; I'd prefer to define it at the top of the function than in the loop. Meant to delete the loop declaration.

Comment (mattpardee) [inline index.js:133]: No prob, :shipit:

Comment (brettshollenberger) [inline index.js:133]: If that's not your style, we can do the loop version

**Pull request merged**